README.md

```diff
@@ -1,22 +1,103 @@
 # Celeste on iOS
 
-## what you'll need
-- an **FNA** build of Celeste (this repo has been tested with the Steam version only, the itch.io version *should* work too)
-- MacOS (if you don't have access to a MacOS machine, you can use a MacOS VM)
-- XCode
-- Visual Studio for Mac with the C#, .NET and Xamarin.iOS workloads
-- patience
-
-## how to build
-- install the needed software mentioned above
-- run `./build.sh`
-- follow the instructions
-- hope it works :3
-
-these instructions might be slightly wrong or have missing information because i wrote this like 3 months ago and don't remember every detail - if you have issues feel free to open an issue :3
-
-## installing
-the script will tell you where the final .ipa file is located - just sideload it to your iOS device using something like TrollStore, AltStore or similar
+# How to Build
+
+This guide will walk you through building a Celeste IPA for iOS using macOS. Please note that **only macOS** is supported for this process. If you don't have a Mac you'll have to use a Virtual Machine.
+
+## Prerequisites
+
+1. **Install Xcode**  
+   - Make sure you have Xcode installed and set up correctly. You can download it from the [Mac App Store](https://apps.apple.com/us/app/xcode/id497799835).
+   - Install the iOS Simulator in Xcode
+
+2. **Install Git**  
+   If Git isn't already installed, running `git` from your terminal will often prompt to install it.
+
+3. **Install FMOD Engine**  
+   - Go to the FMOD download page: [FMOD v1.10.09](https://www.fmod.com/download?version=1.10.09#modengine).  
+   - You will need to create a free FMOD account to download.
+   - Make sure that the FMOD version is set to **v1.10.09 (unsupported)**
+   - Download the **iOS** version of **FMOD Engine**
+   - After downloading, mount the downloaded `fmodstudioapi11009ios-installer.dmg` file by double-clicking it. **No need to do anything further**; simply ensure it's mounted.
```

Comment:
the script already tells you to do this, I'd like to avoid duplicate instructions

Reply:
I just put this in because it felt right to include it in the Prerequisites section, and it also means that if the user was to follow the guide top to bottom, after running the build script they don't have to do anything outside the script, also just wanted to add that all you need to do is mount it.

But yeah I do agree it's better to not have duplicated instructions so I'm happy to remove it, and maybe add "the nothing further to do once mounting" thing in the script? Up to you